.vscode/tasks.json

```diff
@@ -196,19 +196,6 @@
 				}
 			}
 		},
-		{
```

Comment:
Should be extended with the script / task to "Ensure Prerequisites" to provide mechanism of the update of the tools even if new image will not be pulled by developer. This state is true for all runtime dependencies (local, k3d etc)

Reply:
install_prerequisites.sh is added again (but we also don't have a task to update other dependencies, so will also not add one for that)

Reply:
Please create a task to not forget those points

Reply:
which points?